sashimi-webapp/test/unit/specs/database/generated-data/sqlArray.spec.js

```diff
@@ -0,0 +1,31 @@
+import SqlArray from 'src/database/generated-data/sqlArray';
+
+const alasqlArray = new SqlArray();
+
+describe('sqlArray.js', () => {
+  describe('creation of datastructure for alasql', () => {
+    it('should create an empty object of object', () => {
+      alasqlArray.initializeAlasqlArray();
+      const alaArray = alasqlArray.endAlasqlArray();
+      expect(alaArray).to.deep.equal([{}]);
+      expect(typeof alaArray).to.equal('object');
+      expect(typeof alaArray[0]).to.equal('object');
+      /* eslint-disable */
+      expect(typeof Object.values(alaArray[0])).to.equal.null;
+      /* esline-enable */
+    });
+
+    it('should fill up an object of object', () => {
+      alasqlArray.initializeAlasqlArray();
+      alasqlArray.addKeyBasePair('a', 123)
+      alasqlArray.addKeyBasePair('b', '123')
+      const alaArray = alasqlArray.endAlasqlArray();
+      expect(alaArray).to.deep.equal([{a: 123, b: '123'}]);
+      expect(typeof alaArray).to.equal('object');
+      expect(typeof alaArray[0]).to.equal('object');
+      /* eslint-disable */
+      expect(typeof Object.values(alaArray[0])).to.equal.null;
```

Comment:
http://stackoverflow.com/questions/18808226/why-is-typeof-null-object

Unfortunately, `typeof null` returns `"object"` which has no indication of null.
And `typeof` returns a string [Ref: [typeof](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof)]. 

```javascript
expect('object').to.equal.null;
```
will fail. But it passed the test. hmm..